gscp/push_pull.py

```diff
@@ -9,6 +9,13 @@
 _REMOTE_WORK = "remote contains work"
 
 
+def _extract_remote_output(text: str) -> str:
+    lines = [
+        line for line in text.splitlines(keepends=True) if line.startswith("remote:")
```

Comment:
Interestingly, this `remote:` prefix is not language-dependent!
```shell
$ LANG=de_DE.UTF-8 git push -u origin HEAD
Gesamt 0 (Delta 0), Wiederverwendet 0 (Delta 0), Pack wiederverwendet 0
remote:
remote: Create a pull request for 'test2' on GitHub by visiting:
remote:      https://github.com/selimb/gscp/pull/new/test2
remote:
To github.com:selimb/gscp.git
 * [new branch]      HEAD -> test2
Branch 'test2' folgt nun Remote-Branch 'test2' von 'origin'.
```

Reply:
And this is a god send because I don't want to deal with the git API directly hehe